dist/utilities/border/_border.scss

```diff
@@ -6,12 +6,12 @@
 
 @mixin border(
     $important: false,
-    $border: if(variable-exists(border), $border, null)
```

Comment:
Why is this being changed to use $border-color?

Given the mixin name is `border` and not `border-color`, does this not seem misleading?